nemo/collections/asr/parts/manifest.py

```diff
@@ -88,6 +113,8 @@ def __parse_item(line: str, manifest_file: str) -> Dict[str, Any]:
             f"Manifest file {manifest_file} has invalid json line structure: {line} without proper audio file key."
         )
     item['audio_file'] = expanduser(item['audio_file'])
+    if item['audio_file'][0] != "/":
```

Comment:
Forward slash is not always the path seperator, use os.sep everywhere.

Reply:
fixed by isabs()